sapi/cli/php_cli.c

```diff
@@ -588,6 +588,14 @@ BOOL WINAPI php_cli_win32_ctrl_handler(DWORD sig)
 #endif
 /*}}}*/
 
+static int zend_ini_entry_cmp(Bucket *a, Bucket *b)
+{
+	zend_ini_entry *A = Z_PTR(a->val);
+	zend_ini_entry *B = Z_PTR(b->val);
+
+	return zend_binary_strcasecmp(ZSTR_VAL(A->name), ZSTR_LEN(A->name), ZSTR_VAL(B->name), ZSTR_LEN(B->name));
```

Comment:
Do we care about the "order" here? If not you could just use `zend_string_equals`

Reply:
> Do we care about the "order" here?

Yes, I'm intentionally sorting the list to make it independent of the order the extensions are loaded / the extensions define the INI entries.